lms/envs/common.py

```diff
@@ -947,6 +947,18 @@
     # .. toggle_target_removal_date: 2021-10-01
     # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MICROBA-1405'
     'ENABLE_V2_CERT_DISPLAY_SETTINGS': False,
+
+    # .. toggle_name: ENABLE_CODEJAIL_REST_SERVICE
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set this to True if you want to run Codejail code using
+    #   a separate VM or container and communicate with edx-platform using REST API.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2021-08-19
+    # .. toggle_target_removal_date: None
+    # .. toggle_warnings:
+    # .. toggle_tickets:
```

Comment:
Empty annotations here can be removed.

Reply:
Oh... actually, this whole block can be removed, since it's already documented and given a default via the SettingDictToggle.